samza-kafka/src/main/scala/org/apache/samza/checkpoint/kafka/KafkaCheckpointManager.scala

```diff
@@ -219,16 +221,17 @@ class KafkaCheckpointManager(
 
         val messages: util.List[IncomingMessageEnvelope] = envelopes.get(ssp)
         val messagesNum = if (messages != null) messages.size else 0
-        info("CheckpointMgr read %s envelopes (%s messages) from ssp %s. Current offset is %s, newest is %s"
+        debug("CheckpointMgr read %s envelopes (%s messages) from ssp %s. Current offset is %s, newest is %s"
```

Comment:
While we're at it, let's fix this message too:

`"Read % messages from ssp %s. Current offset is %s, newest is %s", format(messagesNum, ssp, currentOffset, newestOffset)`
Note the order of params.